src/platforms/javascript/guides/nextjs/manual-setup/custom-init.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Custom initialialization
+sidebar_order: 3
+description: "Learn how to customize the SDK initialization."
+---
+
+Unlike other Sentry SDKs, the Next.js SDK doesn’t have to be initialized in code, since we do it for you.
+You can still customize the `Sentry.init` call, though.
+
+## Serializable values
+
+Serializable values must be placed in the `next.config.js`:
+
+- `module.exports.serverRuntimeConfig.sentry` for server-side code.
+- `module.exports.publicRuntimeConfig.sentry` for client-side code and fallback for server-side code.
+
+See an example setting `debug: true` in [Manual configuration](/platforms/javascript/guides/nextjs/manual-setup/manual-config#extend-the-usage-of-webpack).
+
+## Non-serializable values
+
+Sentry’s plugin for Next.js, installed along with the SDK, contains the object where non-serializable values must be defined. This must be done in the override of the `App` component:
```

Comment:
@iker-barriocanal do you think it will be clear for the audience of this documentation what is a serializable configuration and what is not?